src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -740,4 +740,15 @@ default AABB getSweepHitBox(@Nonnull ItemStack stack, @Nonnull Player player, @N
     {
         return target.getBoundingBox().inflate(1.0D, 0.25D, 1.0D);
     }
+
+    /**
+     * Used to test if this ItemStack contains the item in question.
+     *
+     * @param stack ItemStack in the Chest slot of the entity.
```

Comment:
Pretty sure that `param` doc is incorrect.

Reply:
Updated the doc in IForgeItem